package.json

```
@@ -140,7 +140,11 @@
   "lint-staged": {
     "*.js": [
       "prettier --write",
+      "eslint --max-warnings=0",
```

```suggestion
      "eslint --max-warnings=0 --fix",
```

Maybe ESLint could fix errors.

---

Nice. Thanks! 😊